v4l2loopback doesn't build with latest kernel

```
substituteStream(): WARNING: pattern '/nix/store/v2x1ks46fq3yn8c94qfv38wrvmr5xn5r-bash-4.4-p23/bin/sh' doesn't match anything in file '/nix/store/v8gyi31b1ky58w2j38y8vxzk62vzgi1m-udev-rules/95-dm-notify.rules'
substituteStream(): WARNING: pattern '/nix/store/h6c0sb2fn70h4n33c7d3znr5bzakrzn6-systemd-243/bin/udevadm' doesn't match anything in file '/nix/store/v8gyi31b1ky58w2j38y8vxzk62vzgi1m-udev-rules/95-dm-notify.rules'
  CC [M]  /build/source/v4l2loopback.o
/build/source/v4l2loopback.c: In function 'vidioc_qbuf':
/build/source/v4l2loopback.c:1516:4: error: implicit declaration of function 'do_gettimeofday'; did you mean 'do_settimeofday64'? [-Werror=implicit-function-declaration]
    do_gettimeofday(&b->buffer.timestamp);
    ^~~~~~~~~~~~~~~
    do_settimeofday64
cc1: some warnings being treated as errors
make[3]: *** [/nix/store/m6yyg2mmw5p8xv71irhcvinqpj9748hh-linux-5.3.7-dev/lib/modules/5.3.7/source/scripts/Makefile.build:281: /build/source/v4l2loopback.o] Error 1
make[2]: *** [/nix/store/m6yyg2mmw5p8xv71irhcvinqpj9748hh-linux-5.3.7-dev/lib/modules/5.3.7/source/Makefile:1626: _module_/build/source] Error 2
make[1]: *** [/nix/store/m6yyg2mmw5p8xv71irhcvinqpj9748hh-linux-5.3.7-dev/lib/modules/5.3.7/source/Makefile:179: sub-make] Error 2
make[1]: Leaving directory '/nix/store/m6yyg2mmw5p8xv71irhcvinqpj9748hh-linux-5.3.7-dev/lib/modules/5.3.7/build'
make: *** [Makefile:43: v4l2loopback.ko] Error 2
builder for '/nix/store/69n689ibilj5gwvlzj0kyhmn762qd9rm-v4l2loopback-0.12.0-5.3.7.drv' failed with exit code 2
cannot build derivation '/nix/store/2vzgmsl136ywkj8svgiykb4wcbm1agg4-kernel-modules.drv': 1 dependencies couldn't be built
cannot build derivation '/nix/store/ac1n2pjb68nhld5vcn1gvy2lzw9pg51s-linux-5.3.7-modules.drv': 1 dependencies couldn't be built
cannot build derivation '/nix/store/zdnhc0nvnh98nsh9cmg881mck1jy11xp-nixos-system-xeep-20.03.git.git.drv': 1 dependencies couldn't be built
error: build of '/nix/store/zdnhc0nvnh98nsh9cmg881mck1jy11xp-nixos-system-xeep-20.03.git.git.drv' failed
```

Will send fix momentarily.

--- a/pkgs/os-specific/linux/v4l2loopback/default.nix
+++ b/pkgs/os-specific/linux/v4l2loopback/default.nix
@@ -2,13 +2,13 @@
 
 stdenv.mkDerivation rec {
   name = "v4l2loopback-${version}-${kernel.version}";
-  version = "0.12.0";
+  version = "0.12.2";
 
   src = fetchFromGitHub {
     owner = "umlaeute";
     repo = "v4l2loopback";
     rev = "v${version}";
-    sha256 = "1rf8dvabksxb2sj14j32h7n7pw7byqfnpqs4m4afj3398y9y23c4";
+    sha256 = "1bcypfz5nlfmdm2a00yl7rgl0jh0g2nmwndxlsrblqclznhjilg2";
   };
 
   hardeningDisable = [ "format" "pic" ];

